exporters/geneva/example/example_metrics.cc

```diff
@@ -26,9 +25,11 @@ const std::string kUnixDomainPath = "/tmp/ifx_unix_socket";
 const std::string kNamespaceName = "test_ns";
 
 void initMetrics(const std::string &name, const std::string &account_name) {
-  std::string conn_string = "Endpoint=unix://" + kUnixDomainPath +
-                            ";Account=" + account_name +
-                            ";Namespace=" + kNamespaceName;
+  std::string conn_string =
```

Comment:
Move this under `#else` instead of assign `conn_string` twice for Windows?

Reply:
Thank fixed.